src/compas_eve/ghpython/components/Ce_Subscribe/code.py

```diff
@@ -0,0 +1,64 @@
+"""
+Subscribe to a topic to receive messages.
+
+COMPAS EVE v0.5.0
+"""
+
+from ghpythonlib.componentbase import executingcomponent as component
+
+from compas_eve import Topic
+from compas_eve import Subscriber
+from compas_eve.ghpython import BackgroundWorker
+
+
+class SubscribeComponent(component):
+    def RunScript(self, transport, topic_name, start, on):
+        if not topic_name:
+            raise ValueError("Please specify the name of the topic")
```

Comment:
the component is red when placed on the canvas, it could be instead orange until all necessary inputs are provided, similar to https://github.com/gramaziokohler/compas_timber/blob/main/src/compas_timber/ghpython/components/CT_Beam_fromCurve/code.py